web/src/beta/lib/core/Map/Layers/hooks.ts

```diff
@@ -561,6 +565,11 @@ export default function useHooks({
     setOverridenLayers(layers => layers.filter(l => !deleted.includes(l.id)));
   }, [atomMap, layers, layerMap, lazyLayerMap, setOverridenLayers, showLayer]);
 
+  useEffect(() => {
+    if (!requestingRenderMode || requestingRenderMode.current === -1) return;
+    requestingRenderMode.current = 1;
```

Comment:
It's really simple rather than the way I expected!!

Reply:
Yes i think this can work well for most of the usecases, but need to take care of none-resource related async updates. In this case the requested render could happen before all async task finishes, and that's why i added a request render from engine API after compute features finish for 3d tiles.

Reply:
I remembered that. I think we need to invoke requestRender after the following lines
- Resource feature(This attaches the style to CZML or KML's features)
  - https://github.com/reearth/reearth/blob/ad4ae6df063b73641bd001a70d8767c53f10e42a/web/src/beta/lib/core/engines/Cesium/Feature/Resource/index.tsx#L81C51-L81C51
  - https://github.com/reearth/reearth/blob/ad4ae6df063b73641bd001a70d8767c53f10e42a/web/src/beta/lib/core/engines/Cesium/Feature/Resource/index.tsx#L146-L148
- Also I guess the clipping box is not working
  -  I think we need to invoke `requestRender` after every `updateTrs` function like [here](https://github.com/reearth/reearth/blob/ad4ae6df063b73641bd001a70d8767c53f10e42a/web/src/beta/lib/core/engines/Cesium/Feature/Box/hooks/box.ts#L81).
  - But if you are not sure about this part, I can handle it. Let me know:pray:

Reply:
Updated.
Note there's some perfermance issue around clipping box.